spy_terminal.py

```diff
@@ -27,76 +30,94 @@ def main():
     while x != 'boom':
```

Comment:
Generic variable names is always a poor practice in programming since it takes away from the code's readability. Use a name describes what `x` is, something like `user_action`